clang-ocl.in

```diff
@@ -2,9 +2,10 @@
 
 set -e
  
-OPENCL_INCLUDE=@OPENCL_INCLUDE@
 CLANG_BIN=@CLANG_BIN@
 BITCODE_DIR=@BITCODE_DIR@
+OPENCL_INCLUDE=`echo | ${CLANG_BIN}/clang -v -x c++ -c - |& grep clang | tail -n1`
+echo $OPENCL_INCLUDE
```

Comment:
I dont think we need to `echo` this.

Reply:
sorry it was an artifact from debugging, just removed it